src/django/api/signals.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import json
+
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from opensearchpy.exceptions import ConnectionError
+
+from api.models.facility.facility import Facility
+from api.services.opensearch import OpenSearchServiceConnection
+from oar.rollbar import report_error_to_rollbar
+
+
+# Initialize logger.
+log = logging.getLogger(__name__)
+
+
+@receiver(post_delete, sender=Facility)
+def location_post_delete_handler_for_opensearch(sender, **kwargs):
+    opensearch = OpenSearchServiceConnection()
+    location_instance = kwargs.get('instance')
+    try:
+        response = opensearch.client.delete(
+            index='production-locations',
```

Comment:
In the future I'd like to have these index values listed in a separate file as constants.